operator-wg/whitepaper-community-review/README.md

```diff
@@ -0,0 +1,882 @@
+While the managed application should provide the telemetry data for itself, the operator could provide metrics about its own behavior and only provides a high level overview about the applications state (as it would be possible for auto-remediation). Furthermore, typical telemetry data provided by the operator could be the count of remediation actions, duration of backups, but also information about the last errors or operational tasks which were handled.
+
+
+#### Scaling (Operator Supports Scaling)
+Scaling is part of the day-2 operations that an operator can manage in order to keep the application / resources functional. The scaling capability doesn’t require the scaling to be automated, but only that the operator will know how to change the resources in terms of horizontal and vertical scaling.
+
+An operator should be able to increase or decrease any resource that it owns, such as CPU, memory, disk size and number of instances.
+
+Ideally the scaling action will be without a downtime. Scaling action ends when all the resources are in consistent state and ready to be used, so an operator should verify the state of all the resources and report it.
+
+#### Auto-Scaling
+An operator should be able to perform the scaling capability based on metrics that it collects constantly and according to thresholds. An operator should be able to automatically increase and decrease every resource that it’s own.
+
+An operator should respect basic scaling configuration of min and max.
+
+
+#### Auto-configuration tuning
+This capability should empower the operator to manage the configuration of the managed application. As an example, the operator could adopt memory settings of an application according to the operation environment (e.g. kubernetes) or the change of DNS names. Furthermore, the operator should be able to handle configuration changes in a seamless way, e.g. if a configuration change requires a restart, this should be triggered. 
+
+These capabilities should be transparent to the users the user should have the possibility to override such auto-configuration mechanisms if they want to do so. Furthermore, automatic reconfigurations should be well-documented in a way that the user could comprehend what is happening on the infrastructure.
+
+#### Uninstalling / Disconnect
+When deleting the declarative requested state (in most cases a custom resource), an operator should allow two behaviors:
+- Uninstalling: An operator should be able to completely remove or delete every managed resource.
+- Disconnecting: An operator should stop managing the provisioned resources.
+
+Both processes should be applied to every resource that the operator directly provisioned.  
+An operator should report any failure in the process in a declarative way (using the [status field](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#object-spec-and-status) for example).
+
+## Security
+![operator model](img/04_1_operator_model.png)
```

Comment:
This diagram is very difficult to read, I suggest redrawing it in excalidraw.com

Reply:
I agree with the sentiment, this hand drawn diagram is tough.

As an alternative to excalidraw.com, I've used draw.io a lot (which I love because you can check in the code for the image to GitHub as it's a text representation - as well as having a great VS Code extension - I don't work for Draw.io, just a happy user!)

Reply:
We will make this drawing a bit easier to read ...